packages/app/tests/acceptance/print-views-test.js

```diff
@@ -15,7 +15,7 @@ module('Acceptance | print views', function(hooks) {
     assert.dom('.navi-report__title').hasText('Kart Wins By Character', 'The report title shows up');
     assert.deepEqual(
       [...document.querySelectorAll('.line-chart-widget .c3-legend-item')].map(el => el.textContent),
```

Comment:
Pretty sure I wrote this, should probably just use `findAll`
```suggestion
      findAll('.line-chart-widget .c3-legend-item').map(el => el.textContent),
```